src/Agda2Hs/Compile/Term.hs

```diff
@@ -195,11 +195,9 @@ compileTerm v = do
       True  -> compileErasedApp es
       False -> (`app` es) . Hs.Con () =<< hsQName (conName h)
     Lit l -> compileLiteral l
-    Lam v b | usableModality v, getOrigin v == UserWritten -> do
-      unless (visible v) $ genericDocError =<< do
-        text "Implicit lambda not supported: " <+> prettyTCM (absName b)
+    Lam v b | keepArg v, getOrigin v == UserWritten -> do
       hsLambda (absName b) <$> underAbstr_ b compileTerm
```

Comment:
Previously we gave an error on a hidden non-erased argument, but with this change we silently drop it. Is this intended?

Reply:
Yeah this is intended and is actually already the behaviour for implicit *arguments*. The typing rules enforce that the only implicit arguments and implicit lambdas that we can write are of type `Level`/`Set`/`Size`.